src/components/TxNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { abbreviateString } from 'utils'
+import { Network } from 'types'
+import { useWalletConnection } from 'hooks/useWalletConnection'
+
+interface TxNotificationProps {
+  txHash: string
+}
+
+function getDomainPrefix(networkId: Network): string {
```

Comment:
move to utils?

Reply:
I decided to move it back because it's a pretty contained function. I don't see where this would be reused other than on the new component.